2025-10-07 02:38:10,170 CRIT Supervisor is running as root.  Privileges were not dropped because no user is specified in the config file.  If you intend to run as root, you can set user=root in the config file to avoid this message.
2025-10-07 02:38:10,174 INFO supervisord started with pid 1
2025-10-07 02:38:11,177 INFO spawned: 'nginx' with pid 7
2025-10-07 02:38:11,180 INFO spawned: 'php-fpm' with pid 8
2025-10-07 02:38:11,231 CRIT uncaptured python exception, closing channel <POutputDispatcher at 131647799786720 for <Subprocess at 131647799879440 with name php-fpm in state STARTING> (stderr)> (<class 'OSError'>:[Errno 29] Illegal seek [/usr/lib/python3/dist-packages/supervisor/supervisord.py|runforever|218] [/usr/lib/python3/dist-packages/supervisor/dispatchers.py|handle_read_event|276] [/usr/lib/python3/dist-packages/supervisor/dispatchers.py|record_output|210] [/usr/lib/python3/dist-packages/supervisor/dispatchers.py|_log|179] [/usr/lib/python3/dist-packages/supervisor/loggers.py|info|327] [/usr/lib/python3/dist-packages/supervisor/loggers.py|log|345] [/usr/lib/python3/dist-packages/supervisor/loggers.py|emit|227] [/usr/lib/python3/dist-packages/supervisor/loggers.py|doRollover|264])
2025-10-07 02:38:11,233 CRIT uncaptured python exception, closing channel <POutputDispatcher at 131647799786720 for <Subprocess at 131647799879440 with name php-fpm in state STARTING> (stderr)> (<class 'OSError'>:[Errno 29] Illegal seek [/usr/lib/python3/dist-packages/supervisor/supervisord.py|runforever|218] [/usr/lib/python3/dist-packages/supervisor/dispatchers.py|handle_read_event|276] [/usr/lib/python3/dist-packages/supervisor/dispatchers.py|record_output|210] [/usr/lib/python3/dist-packages/supervisor/dispatchers.py|_log|179] [/usr/lib/python3/dist-packages/supervisor/loggers.py|info|327] [/usr/lib/python3/dist-packages/supervisor/loggers.py|log|345] [/usr/lib/python3/dist-packages/supervisor/loggers.py|emit|227] [/usr/lib/python3/dist-packages/supervisor/loggers.py|doRollover|264])
2025-10-07 02:38:12,234 INFO success: nginx entered RUNNING state, process has stayed up for > than 1 seconds (startsecs)
2025-10-07 02:38:12,234 INFO success: php-fpm entered RUNNING state, process has stayed up for > than 1 seconds (startsecs)
2025-10-07 02:38:39,800 CRIT uncaptured python exception, closing channel <POutputDispatcher at 131647799786720 for <Subprocess at 131647799879440 with name php-fpm in state RUNNING> (stderr)> (<class 'OSError'>:[Errno 29] Illegal seek [/usr/lib/python3/dist-packages/supervisor/supervisord.py|runforever|218] [/usr/lib/python3/dist-packages/supervisor/dispatchers.py|handle_read_event|276] [/usr/lib/python3/dist-packages/supervisor/dispatchers.py|record_output|210] [/usr/lib/python3/dist-packages/supervisor/dispatchers.py|_log|179] [/usr/lib/python3/dist-packages/supervisor/loggers.py|info|327] [/usr/lib/python3/dist-packages/supervisor/loggers.py|log|345] [/usr/lib/python3/dist-packages/supervisor/loggers.py|emit|227] [/usr/lib/python3/dist-packages/supervisor/loggers.py|doRollover|264])
2025-10-07 02:42:10,802 CRIT uncaptured python exception, closing channel <POutputDispatcher at 131647799786720 for <Subprocess at 131647799879440 with name php-fpm in state RUNNING> (stderr)> (<class 'OSError'>:[Errno 29] Illegal seek [/usr/lib/python3/dist-packages/supervisor/supervisord.py|runforever|218] [/usr/lib/python3/dist-packages/supervisor/dispatchers.py|handle_read_event|276] [/usr/lib/python3/dist-packages/supervisor/dispatchers.py|record_output|210] [/usr/lib/python3/dist-packages/supervisor/dispatchers.py|_log|179] [/usr/lib/python3/dist-packages/supervisor/loggers.py|info|327] [/usr/lib/python3/dist-packages/supervisor/loggers.py|log|345] [/usr/lib/python3/dist-packages/supervisor/loggers.py|emit|227] [/usr/lib/python3/dist-packages/supervisor/loggers.py|doRollover|264])
2025-10-07 02:42:18,428 CRIT uncaptured python exception, closing channel <POutputDispatcher at 131647799786720 for <Subprocess at 131647799879440 with name php-fpm in state RUNNING> (stderr)> (<class 'OSError'>:[Errno 29] Illegal seek [/usr/lib/python3/dist-packages/supervisor/supervisord.py|runforever|218] [/usr/lib/python3/dist-packages/supervisor/dispatchers.py|handle_read_event|276] [/usr/lib/python3/dist-packages/supervisor/dispatchers.py|record_output|210] [/usr/lib/python3/dist-packages/supervisor/dispatchers.py|_log|179] [/usr/lib/python3/dist-packages/supervisor/loggers.py|info|327] [/usr/lib/python3/dist-packages/supervisor/loggers.py|log|345] [/usr/lib/python3/dist-packages/supervisor/loggers.py|emit|227] [/usr/lib/python3/dist-packages/supervisor/loggers.py|doRollover|264])
2025-10-07 02:42:41,179 CRIT uncaptured python exception, closing channel <POutputDispatcher at 131647799786720 for <Subprocess at 131647799879440 with name php-fpm in state RUNNING> (stderr)> (<class 'OSError'>:[Errno 29] Illegal seek [/usr/lib/python3/dist-packages/supervisor/supervisord.py|runforever|218] [/usr/lib/python3/dist-packages/supervisor/dispatchers.py|handle_read_event|276] [/usr/lib/python3/dist-packages/supervisor/dispatchers.py|record_output|210] [/usr/lib/python3/dist-packages/supervisor/dispatchers.py|_log|179] [/usr/lib/python3/dist-packages/supervisor/loggers.py|info|327] [/usr/lib/python3/dist-packages/supervisor/loggers.py|log|345] [/usr/lib/python3/dist-packages/supervisor/loggers.py|emit|227] [/usr/lib/python3/dist-packages/supervisor/loggers.py|doRollover|264])
2025-10-07 02:42:41,217 CRIT uncaptured python exception, closing channel <POutputDispatcher at 131647799786720 for <Subprocess at 131647799879440 with name php-fpm in state RUNNING> (stderr)> (<class 'OSError'>:[Errno 29] Illegal seek [/usr/lib/python3/dist-packages/supervisor/supervisord.py|runforever|218] [/usr/lib/python3/dist-packages/supervisor/dispatchers.py|handle_read_event|276] [/usr/lib/python3/dist-packages/supervisor/dispatchers.py|record_output|210] [/usr/lib/python3/dist-packages/supervisor/dispatchers.py|_log|179] [/usr/lib/python3/dist-packages/supervisor/loggers.py|info|327] [/usr/lib/python3/dist-packages/supervisor/loggers.py|log|345] [/usr/lib/python3/dist-packages/supervisor/loggers.py|emit|227] [/usr/lib/python3/dist-packages/supervisor/loggers.py|doRollover|264])
2025-10-07 02:43:13,491 CRIT uncaptured python exception, closing channel <POutputDispatcher at 131647799786720 for <Subprocess at 131647799879440 with name php-fpm in state RUNNING> (stderr)> (<class 'OSError'>:[Errno 29] Illegal seek [/usr/lib/python3/dist-packages/supervisor/supervisord.py|runforever|218] [/usr/lib/python3/dist-packages/supervisor/dispatchers.py|handle_read_event|276] [/usr/lib/python3/dist-packages/supervisor/dispatchers.py|record_output|210] [/usr/lib/python3/dist-packages/supervisor/dispatchers.py|_log|179] [/usr/lib/python3/dist-packages/supervisor/loggers.py|info|327] [/usr/lib/python3/dist-packages/supervisor/loggers.py|log|345] [/usr/lib/python3/dist-packages/supervisor/loggers.py|emit|227] [/usr/lib/python3/dist-packages/supervisor/loggers.py|doRollover|264])
